src/main/java/org/kainos/groupA/models/Responsibility.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.groupA.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Responsibility {
+
+    private String role_name;
```

Comment:
the role name is always going to be the same in the list right? In that case our object should be:
{
private String role_name;
private List<String> resp_desc;
}

and instead of returning a list of type Responsibility from the DAO you can just return an object of type Responsibility